sycl/ReleaseNotes.md

```diff
@@ -10,7 +10,8 @@ Release notes for commit range 2ffafb95f887..6a49170027fb
  - [ESIMD] Added indirect read and write methods to ESIMD class [8208427]
  - Provided `sycl::ONEAPI::has_known_identity` type trait to determine if
    reduction interface supports user-defined type at compile-time [0c7bd24]
-   [060fd50]
+   [060fd50]. Note: `sycl::ONEAPI::` is deprecated, please use `sycl::ext::oneapi::`
```

Comment:
What is the point of editing old release notes?

I think that deprecation notice should appear in the new section for the next release instead of being hidden in already published notes.

Tagging @romanovvlad and @bader for their feedback

Reply:
Yes, I also revised this - this is not a good idea because we add changes from newer release to the older release, the only pros is that if customer uses newer version of the compiler where we removed sycl::ONEAPI and wanted to see what features can be used from older releases, they will be confused after finding out that e.g., there is no sycl::ONEAPI::has_known_identity in the product.
I will revert this.

Reply:
Done